core/domain/wipeout_service.py

```diff
@@ -376,6 +377,24 @@ def _delete_models_with_delete_at_end_policy(user_id: str) -> None:
             model_class.apply_deletion_policy(user_id)
 
 
+def _delete_profile_picture(username: str) -> None:
+    """Verify that the profile picture is deleted.
+
+    Args:
+        username: str. The username of the user.
+    """
+    if username is None:
+        return
```

Comment:
Why is this needed?

Reply:
username can be None to prevent that part I added this and it was also creating problems for lots of test in the test file.

Reply:
But then the type is wrong.

Reply:
This code was updated, this is the old comment that is why I think. I removed this check from here and added where we are calling this function.